dotcom-rendering/src/types/content.ts

```diff
@@ -53,6 +53,7 @@ export interface CalloutBlockElement {
 	id: string;
 	calloutsUrl: string;
 	activeFrom: number;
+	activeUntil?: number;
```

Comment:
Does this have to be optional - any idea @marjisound @abeddow91 @sophie-macmillan  ?